code/src/Card.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import CoverImage from './Cover-image'
+import AlbumName from './Album-name'
+import ArtistName from './Artist-name'
+import data from './data.json';
+import AlbumCoverOverlay from './Album-cover-overlay'
+
+const albumsArray = data.albums.items;
+
+const Card = () => {
+    return (
+        <>
```

Comment:
If your component returns only one root element, there is no need to wrap it around React Fragment.

In this case WE NEED to use React Fragment :

```jsx
return (
	<div>My element 1</div>
        <div>My element 2</div>
);
```

so, after necessary changes it looks like this :

```jsx
return (
	<>
		<div>My element 1</div>
	        <div>My element 2</div>
	</>
);
```

But in this case WE DON'T NEED to use React Fragment :

```jsx
return (
	<div>My element 1</div>
);
```